java/src/jmri/jmrit/Bundle.properties

```diff
@@ -44,7 +44,7 @@ MenuItemPowerControl = Power Control
 MenuItemSensorGroup = Sensor Groups...
 MenuItemSpeedometer = Speedometer
 MenuItemMeter = Voltage / Current meter
-MenuItemConsistTool = Consist Control
+MenuItemConsistTool = Consist Tool
```

Comment:
If the menu item has changed from `Consist Control` to `Consist Tool`, it should be mentioned in the release notes.

Reply:
ok, done, will be in next push.